### ISSUE TITLE
Multiple scroll areas

### ISSUE THREAD

**Issue by grommas** [ashthornton/asscroll#61]
Hi @ashthornton

I’m fiddling around with your scroller (again). Is it possible to have multiple scroll areas? It would be nice to have separated and smoothly scrollable areas. My current tests always have some glitches. 

[Codepen: Multiple scroll areas (test 1: one scroller)](https://codepen.io/grommas/pen/ExmoMwW?editors=0010)
[Codepen: Multiple scroll areas (test 2: multiple scrollers)](https://codepen.io/grommas/pen/JjNpRxz?editors=0010)

Any advice?

**Comment by ashthornton** [comment]
This is something I have thought about adding before. It would be nice to have the option to create multiple instances and target multiple elements

But right now as you can see it would involve some really hacky methods in order to get this to work. The main issue is that ASScroll listens to `wheel` and `scroll` events on the `window` rather than the element you're scrolling, so it will scroll all instances at once. In theory this isn't too difficult to implement so I'll make this an enhancement and get round to it at some point!

**Comment by grommas** [comment]
Makes sense. So I’ll try to find another solution for now. Nevertheless it would be a great enhancement! Thanks for your reply!
